service/src/main/java/bio/terra/drshub/services/DrsApiFactory.java

```diff
@@ -28,33 +24,25 @@
 public class DrsApiFactory {
 
   private static final Integer CONNECTION_POOL_SIZE = 500;
-  private final Map<Pair<String, String>, DrsApi> factoryCache =
-      Collections.synchronizedMap(new HashMap<>());
 
   public DrsApi getApiFromUriComponents(UriComponents uriComponents, DrsProvider drsProvider) {
-    var key = Pair.of(uriComponents.getHost(), drsProvider.getName());
-    return factoryCache.computeIfAbsent(
-        key,
-        pair -> {
-          log.info(
-              "Creating new DrsApi client for host '{}', for DRS Provider '{}'",
-              pair.getLeft(),
-              pair.getRight());
-          var mTlsConfig = drsProvider.getMTlsConfig();
-          var drsClient =
-              mTlsConfig == null
-                  ? new ApiClient(makeRestTemplateWithPooling())
-                  : new ApiClient(
-                      makeMTlsRestTemplateWithPooling(
-                          mTlsConfig.getCertPath(), mTlsConfig.getKeyPath()));
+    log.info(
+        "Creating new DrsApi client for host '{}', for DRS Provider '{}'",
+        uriComponents.getHost(),
+        drsProvider.getName());
+    var mTlsConfig = drsProvider.getMTlsConfig();
+    var drsClient =
```

Comment:
Are we worried about overhead from getting rid of the cache?  I guess we can up replicas if needed?

Reply:
I have concerns too, but this was more of a premature optimization that didn't pan out. The most latency was coming from requests to Bond, and the caches in front of that should be doing the bulk of the speedup. The only thing this would do is cause more heap space to be needed, but we can tune that pretty easily.

Reply:
Yeah, that sounds totally reasonable